Install_Instructions.md

```diff
@@ -0,0 +1,43 @@
+# Tux Typing 1.8.3 Installation Guide
```

Comment:
I suppose I would append this info to the README.

Also add a link or ref to https://github.com/tux4kids/tuxtype/blob/master/doc/INSTALL for those who need to build from source.

Reply:
Sure